gen/eduroam_block.pl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+
+local $::SERVICE_NAME = "eduroam_block";
+local $::PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+
+my $fileName = "$DIRECTORY/$::SERVICE_NAME";
+my $data = perunServicesInit::getFlatData;
+
+#Constants
+
+our $A_LOGIN; *A_LOGIN = \'urn:perun:user_facility:attribute-def:virt:login';
+
+
+# GATHER USERS
+my $users;  # $users->{$login}->{ATTR} = $attrValue;
+
+foreach my $user (($data->getChildElements)[1]->getChildElements) {
+
+    my %uAttributes = attributesToHash $user->getAttributes;
+
+    my $login = $uAttributes{$A_LOGIN};
+
+    $users->{$login} = 1;
+}
+
+open FILE,">$fileName" or die "Cannot open $fileName: $! \n";
+binmode FILE, ":utf8";
+
+
+my @logins = sort keys %{$users};
+for my $login (@logins) {
+
+    print FILE $users->{$login}{$A_LOGIN} . "\tAuth-Type := Reject" . "\n";
```

Comment:
Just print login, since it's key in a hash ``print FILE $login . "\tAuth-Type......``